run-tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
IMO this should be merged into the build script.  It should be exposed with a `--test` option.  This give the build experience a more common feel/

Reply:
Nikola said you were disccussed and the plan is set a new property 'RunTests' which is similar to generate.sh. The test project should only be added if new property 'RunTests' is set. That's why we need this new script.

Reply:
We did discuss this offline and while the original plan (in PR) would have worked, I see the benefit of fitting 'test' experience within the `build.sh` script. It would require some tweaks in `eng/Build.props`.

Reply:
Updated. Now just use build.sh --test to launch the test